memory/test/test_memory.py

```diff
@@ -0,0 +1,42 @@
+import concurrent
+import unittest
+import uuid
+from memory.client import BlockingMemoryClient
+
+from hailtop.config import get_user_config
+from hailtop.google_storage import GCS
+
+
+class FileTests(unittest.TestCase):
```

Comment:
We should have and additional test that verifies users are isolated.

Reply:
i think this is going to be involved enough that I'd like to address it in a separate PR.